apps/svelte.dev/content/blog/2025-08-01-whats-new-in-svelte-august-2025.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "What's new in Svelte: August 2025"
+description: 'Async Svelte is out - with all the fixins'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) officially out in the world and its companion [Remote Functions](https://github.com/sveltejs/kit/discussions/13897) already ligned up, there's a lot of updates to share!
+
+Be sure to check out the showcase for videos from X, Y and Z about all the brand new features in Svelte.
+
+Let's dive into the highlights...
+
+## What's new in Svelte, SvelteKit, Language Tools and CLI (sv)
+
+- `getAbortSignal` returns an `AbortSignal` that aborts when the current derived or effect re-runs or is destroyed (**svelte@5.35.0**, [Docs](https://svelte.dev/docs/svelte/svelte#getAbortSignal), [#16266](https://github.com/sveltejs/svelte/pull/16266))
+- A new `parent` property has been added to the `__svelte_meta` properties that are added to elements at dev time - useful for debugging nested hierarchies (**svelte@5.35.1**, Docs, [#16255](https://github.com/sveltejs/svelte/pull/16255))
+- `await` is now supported in components when using the `experimental.async` compiler option (**svelte@5.36.0/extensions@109.10.0**, [Docs](https://svelte.dev/docs/svelte/await-expressions) [#15844](https://github.com/sveltejs/svelte/pull/15844))
```

Comment:
Remote functions was just released in https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%402.27.0

Docs: https://svelte.dev/docs/kit/remote-functions
PR: https://github.com/sveltejs/kit/pull/13986